test/jdk/java/net/Socks/SocksIPv6Test.java

```diff
@@ -120,7 +126,6 @@ private boolean ensureInet6AddressFamily() throws IOException {
 
     @Test(groups = "unit")
     public void testSocksOverIPv6() throws Exception {
-        if (!shouldRun) return;
```

Comment:
Please remove empty line as in orig change.

Reply:
fixed